Sources/PointFree/PublicEpisodes/0010-ATaleOfTwoFlatMaps.swift

```diff
@@ -1027,7 +1027,7 @@ In just a few lines and in a single expression, we were able to apply a complex
       type: .paragraph
     ),
     Episode.TranscriptBlock(
-      content: "What's the point?",
+      content: "What’s the point?",
```

Comment:
doesn't this already happen with markdown rendering?

Reply:
Titles don't render with Markdown yet :(